lib/msal-common/src/response/AuthenticationResult.ts

```diff
@@ -39,5 +39,6 @@ export type AuthenticationResult = {
     cloudGraphHostName?: string;
     msGraphHost?: string;
     code?: string;
+    tokenSizes?: object;
```

Comment:
I don't think we should expose this on the result object. Should just be used internally.